cmd/frontend/build_frontend_env/build-frontend-env.go

```diff
@@ -0,0 +1,81 @@
+package build_frontend_env
+
+/*
+$ your_cli_tool build-frontend-env --secret "YourSecretName" --bucket "YourBucketName"
+*/
+
+import (
+    "os"
+    "strings"
+    "fmt"
+    "log"
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/secretsmanager"
+    "github.com/spf13/cobra"
+    "github.com/DSGT-DLP/Deep-Learning-Playground/cli/cmd/frontend" // For frontend/
+)
+
+var secretName string // Name of the secret in AWS Secrets Manager.
+var bucketName string // Name of the bucket (Assuming one bucket here).
+
+var buildFrontendEnvCmd = &cobra.Command{
+    Use:   "build-frontend-env",
+    Short: "Build .env file for frontend/",
+    Run: func(cmd *cobra.Command, args []string) {
+        sess, err := session.NewSession(&aws.Config{Region: aws.String("us-west-2")})
+        if err != nil {
+            log.Fatal("Error creating AWS session: ", err)
+        }
+
+        smClient := secretsmanager.New(sess)
+        secretValue, err := smClient.GetSecretValue(&secretsmanager.GetSecretValueInput{SecretId: aws.String(secretName)})
+        if err != nil {
+            log.Fatal("Error retrieving secret: ", err)
+        }
+
+        path := "./frontend"
+
+        // Adding secrets to the .env file
+        writeToEnvFile(secretName, *secretValue.SecretString, path)
+
+        // ** need to add project constants to .env file **
+        writeToEnvFile("PROJECT_CONSTANT", "YourConstantValue", path)
+
+        // ** need to add bucket name to .env file **
+        writeToEnvFile("BUCKET_NAME", bucketName, path)
```

Comment:
we can also write the `dlp-executions-bucket` as a parameter store value. 

`dlp-upload-bucket` is the name of S3 bucket responsible for uploading dataset files to S3

Reply:
oh ok, so you want me to hard-code 'dlp-upload'bucket' here and just write it to that bucket. 

and what about dlp-executions-bucket?

Reply:
`dlp-upload-bucket` and `dlp-executions-bucket` could be stored in parameter store if you'd like, but not required as per my other comment

Reply:
i see ok got it